### PR TITLE
PLAT-825 Implement wrappers for factories and transports (test purpose)

### PR DESCRIPTION
**Pull request by dgovorukhin**

### CODE REVIEW

File: ledger-core/network/nds/uniproto/l2/uniserver/server.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ func NewUnifiedServer(dispatcher uniproto.Dispatcher, logger MiniLogger) *Unifie
 		panic(throw.IllegalValue())
 	}
 	s := &UnifiedServer{logger: logger}
+	s.transportProvider = defaultProvider
```
Comment (AndreyBronin) [inline ledger-core/network/nds/uniproto/l2/uniserver/server.go:54]: its not good idea to use global object

Comment (dgovorukhin) [reply ledger-core/network/nds/uniproto/l2/uniserver/server.go:54]: What the best way for this?

Comment (AndreyBronin) [reply ledger-core/network/nds/uniproto/l2/uniserver/server.go:54]: s.transportProvider = &DefaultTransportProvider{}

**Review by AndreyBronin — APPROVED** [review]

File: ledger-core/network/nds/uniproto/l2/uniserver/server.go (hunk 1 above)
Comment (dgovorukhin) [reply ledger-core/network/nds/uniproto/l2/uniserver/server.go]: Fixed

**Review by MariaVasilenko — APPROVED** [review]

**Pull request merged**